### DISCUSSION TITLE
Mamba3D

### DISCUSSION THREAD

**Discussion by HuayuanSong** [LMD0311/PointMamba#10] [Q&A] [answered]
Hi @LMD0311 ,
I just saw that you have submitted a new paper on 'Mamba3D': https://arxiv.org/pdf/2404.14966

I was just wondering if and when code will be publicly available for that? I would love to reproduce the results :))

Thanks!

**Comment by LMD0311** [comment, marked as answer]
I apologize for the confusion. Although all authors of `Mamba3D` are affiliated with Huazhong University of Science and Technology, we are not the authors of the article 😂. You may want to contact the actual author via email. Here is a link 👉 https://nini-lxz.github.io/ @HuayuanSong

**Reply by HuayuanSong** [reply]
Ah okay, sorry about that, didn't look thoroughly. Thanks for your help and quick response!!
